src/e2e/specs/purchase.spec.ts

```diff
@@ -6,6 +6,11 @@ import { test, expect } from "../fixtures/basePage.js";
 import { checkAuthState, setEnvVariables } from "../utils/helpers.js";
 
 test.describe(`${process.env.E2E_TEST_ENV} - Breach Scan, Monitor Plus Purchase Flow`, () => {
+  test.skip(
```

Comment:
So what's the diff in the # of tests we are running full vs. smoke? 
Since we are still skipping these purchase tests?

Reply:
Do we expect to ever run these tests via PR CI? 
I guess the "full E2E suite" seems misleading / implies that we are running all the tests?

Reply:
These were skipped before as well in the individual tests [here](https://github.com/mozilla/blurts-server/pull/4996/files#diff-ed6beb47e73545f4946cf931849b953f093a55909d8c3fd05b8c84f9800247c3L47-L50). I’ve just moved it to the `describe` block. The tests were set up (not part of this PR) to only work and run on `stage` I believe?

Reply:
Thanks, yea that makes sense!